airbyte-ci/connectors/metadata_service/lib/metadata_service/gcs_upload.py

```diff
@@ -304,6 +342,12 @@ def upload_metadata_to_gcs(bucket_name: str, metadata_file_path: Path, validator
 
     doc_version_uploaded, doc_version_blob_id = _doc_upload(metadata, bucket, docs_path, False, False)
     doc_inapp_version_uploaded, doc_inapp_version_blob_id = _doc_upload(metadata, bucket, docs_path, False, True)
+    manifest_yml_uploaded, manifest_yml_blob_id = upload_file_if_changed(
+        yaml_manifest_file_path, bucket, f"{METADATA_FOLDER}/{metadata.data.dockerRepository}/manifest.yaml"
+    )
+    components_py_uploaded, components_py_blob_id = upload_file_if_changed(
+        yaml_manifest_file_path, bucket, f"{METADATA_FOLDER}/{metadata.data.dockerRepository}/components.py"
+    )
```

Comment:
TODO: probably we want to zip or pack this before uploading. I think otherwise, antivirus programs may be tempted to delete/quarantine the file when it is downloaded.